pkg/cmd/step/e2e/step_e2e_gc.go

```diff
@@ -63,12 +70,61 @@ func NewCmdStepE2EGC(commonOpts *opts.CommonOptions) *cobra.Command {
 	cmd.Flags().StringVarP(&options.Region, "region", "", "europe-west1-c", "GKE region to use. Default: europe-west1-c")
 	cmd.Flags().StringVarP(&options.ProjectID, "project-id", "p", "", "Google Project ID to delete cluster from")
 	cmd.Flags().IntVarP(&options.Duration, "duration", "d", 2, "How many hours old a cluster should be before it is deleted if it does not have a --delete tag")
+	cmd.Flags().StringArrayVarP(&options.Providers, "providers", "", []string{"gke"}, "The providers to run the cleanup for")
```

Comment:
Why default to `gke`? To stay backwards compatible?

Reply:
Yes, I wanted the step to behave exactly the same when not providing EKS as a provider until we consolidate the abstraction for GKE too.

Reply:
ok